docs/UserGuide.md

```diff
@@ -67,21 +67,22 @@ Adds a new friend to Amigos. A friend has:
 * *(optional)* email
 * *(optional)* address
 * *(optional)* description
+* *(optional)* tags
```

Comment:
If comment below is followed, then these lines are no longer necessary because command will already say which fields are compulsory and which aren't.